packages/devtools_app/lib/src/shared/primitives/simple_items.dart

```diff
@@ -50,3 +50,26 @@ const String traceEventsFieldName = 'traceEvents';
 const closureName = '<closure>';
 
 const anonymousClosureName = '<anonymous closure>';
+
+const _memoryDocUrl =
+    'https://docs.flutter.dev/development/tools/devtools/memory';
+const _consoleDocUrl =
+    'https://docs.flutter.dev/development/tools/devtools/console';
+
+enum DocLinks {
```

Comment:
nit: rename to MemoryDocsLinks since these are all for the memory screen

Reply:
then we should rename each enum value to start with 'memory' so that these are more accurately named.